.circleci/config.yml

```diff
@@ -137,14 +255,37 @@ jobs:
     <<: *defaults
     steps:
       - checkout
+      - setup_remote_docker:
+          version: 20.10.2
+          docker_layer_caching: true
       - run:
           name: Install github-release
           command: |
             GO111MODULE=off go get github.com/github-release/github-release
+      - run:
+          name: Install xgo
+          command: |
+            go install -v github.com/pastelnetwork/xgo@48be1f35b934924c998997a8470ffdb6a022b968
+      - create-sources-container:
+          containerName: "sourcesContainer"
       - release:
-          package: "walletnode"
+          buildContainerName: "pqsignatures"
+          moduleSubDir: "pqsignatures"
+          package: "cmd/pqsignatures"
+          targets: "linux/amd64,darwin-10.14/amd64,windows/amd64,"
+          binaryName: "pqsignatures"
       - release:
-          package: "supernode"
+          buildContainerName: "walletnode"
+          moduleSubDir: "walletnode"
+          package: ""
+          targets: "linux/amd64,darwin-10.14/amd64,windows/amd64,"
+          binaryName: "walletnode"
+      - release:
+          buildContainerName: "supernode"
+          moduleSubDir: "supernode"
+          package: ""
+          targets: "linux/amd64,darwin-10.14/amd64,windows/amd64,"
```

Comment:
If we reduce list of the OS, does this cause an error in the `github-upload-all-os:` step ?

Reply:
yes, `github-upload-all-os` will fail :)

Reply:
I've fixed similar issue for `build` command and its `Validate existence of built binaries`

Reply:
Just to clarify, if we change `targets` for supernode
```
targets: "linux/amd64,darwin-10.14/amd64,windows/amd64,"
```
to 
```
targets: "linux/amd64,"
```

in line 163 `github-release upload` will not return error ? when `github-upload-all-os` tries to upload a binary that doesn't exists?

Reply:
@levkoburburas my initial supposing was to use currently removed `release-os` command for linux-only builds.

but yes, uploading will fail if to do mentioned change.

Reply:
@maxhora Understood, if we need it in the future, we will probably have to transfer some part of the config to Makefile